css/css-flexbox/flexbox_justifycontent-left-001-ref.html

```diff
@@ -9,11 +9,9 @@
   width: 40em;
   display: flex;
```

Comment:
Ideally we should avoid using flexbox in the reference, so it's a different codepath.